romanesco/__main__.py

```diff
@@ -23,6 +23,7 @@ def run(*pargs, **kwargs):
                         url=jobInfo.get('url'), method=jobInfo.get('method'),
                         headers=jobInfo.get('headers')) as jm:
             kwargs['_job_manager'] = jm
```

Comment:
Ok. Not sure it should be prefixed here or in the the script env - e.g. we inject the Spark context as `sc` with no underscore in the spark plugin. I'll create a separate issue to address this - I wouldn't change anything in this PR.